AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -323,8 +323,12 @@ - (BOOL)configureWithAppSecret:(NSString *)appSecret
 }
 
 - (void)start:(NSString *)secretString withServices:(NSArray<Class> *)services fromApplication:(BOOL)fromApplication {
-  NSString *appSecret = [MSUtility appSecretFrom:secretString];
-  NSString *transmissionTargetToken = [MSUtility transmissionTargetTokenFrom:secretString];
+  NSString *appSecret;
+  NSString *transmissionTargetToken;
+  if (secretString) {
```

Comment:
Can we adjust parameters nullability instead?

Reply:
Fixed